### PR TITLE
[P3 tests] [Team 14] Phrase Search Tests

### PR DESCRIPTION
**Pull request by omidkv**
What's your team number?
Team 14

What is the functionality being tested?
Phrase Search

Describe your tests briefly:
test1 tests phrase search when all the documents are on a single segment. This is checked against multiple different phrases.

test2 tests to see if the phrase isn't there that hasNext will return false right away

test3 tests phrase search when each of the documents is on a separate segment. For this test, we only check a single phrase.

Does each test case have comments/documentation?
Yes

### CODE REVIEW

File: src/test/java/edu/uci/ics/cs221/index/positional/Team14PhraseSearchTest.java [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+        index.delete();
+        InvertedIndexManager.DEFAULT_FLUSH_THRESHOLD = 1000;
+        InvertedIndexManager.DEFAULT_MERGE_THRESHOLD = 8;
+    }
+
+
+    /*
+
+        For this test we check to make sure that if all of the documents are on one segment that we can correctly find
+        all of the phrases.
```
Comment (sadeemsaleh) [inline src/test/java/edu/uci/ics/cs221/index/positional/Team14PhraseSearchTest.java]: But the default flush threshold is set to 1 so without explicitly calling flush each document is in a different segment

**Comment by zerowei** [comment]
Just like the review of team 12, there is something wrong with the description of test 1 because you have already set the flush threshold to 1 in the build() function. So it's better to revise it although it doesn't change the result. Besides, I think you can also test if the search returns null when the sequence of keywords is not consecutive.

Team 11
